js/apps/system/_admin/aardvark/APP/react/src/setupProxy.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
+/* 
+ * Proxy Configuration (see routes array for details)
+ */
+
+const ARANGODB_PORT = 8529;
+
+const routes = [
```

Comment:
Can you tell someone like me who has no idea about this why this list is complete?

Reply:
Before, the proxy was defined to proxy all requests. That lead to also fetch the "build files" the foxx application aardvark is delivering. That are only the heavily generated and minified/uglified files etc. Now we're only allowing that `routes` to being proxied which are defined here. We want to read all the images via aardvark and of course the API, as we do not have an API on the react http server side. Same goes for statistics. It could be possible that this list is not complete. But we'll see errors during development. And those potential errors are only in dev mode and not production.